### DISCUSSION TITLE
[Art competition] safety & liveness property

### DISCUSSION THREAD

**Discussion by Yeonwoo-Nam** [prosyslab-classroom/cs524-program-analysis#282] [Art competition]
** Upload your picture and provide the following description. Delete this line. **
![sky slope](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/69100141/d111bd5a-38b8-4567-b96b-bee298f76c44)

## Name
남연우

## Tool You Used
Bing Image Creator

## Short Description
For me, [safety & liveness property] means [back of a man with pole starting go down at sky slope with huge fence faraway, where some are already going down].

## Long Description (up to 3 sentences)
The man has safety in the sense that he will never go out of the fence no matter how far he goes down. 
Also, He has liveness in the sense that he will eventually reach the endpoint, which is the lowest point, if he let the gravity take him to a strictly lower point whenever time passes.
Each person in the slope behaves differently, but we can ensure that they all have safety & liveness properties though we don't know the exact shape of the whole slope.

**Comment by KihongHeo** [comment]
Interesting. Nice try!

**Comment by KihongHeo** [comment]
Hi. Did you reduce the size of this pic? Can you upload the original version if possible?

**Comment by Yeonwoo-Nam** [comment]
![sky slope](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/69100141/e2fceb4b-5e13-4f73-8ea0-ccca4638850a)
This version is 1024*1024.
